### DISCUSSION TITLE
Main Menu Theme

### DISCUSSION THREAD

**Discussion by conroy-tafesa** [conroy-tafesa/T24S2T1EXTG03#1] [General]
Are we happy with the current theme for the main menu page? I'm inclined to suggest using a lighter colour for the buttons to make them a little more readable.

**Comment by rajeetdon** [comment]
I am happy with the colour. Yes, you can make light color for the button.

**Reply by conroy-tafesa** [reply]
Excellent! I'll put that change in my next commit.

**Comment by rajeetdon** [comment]
I am happy with the colour. Yes you can make light color for the button.
